src/conn/conn_api.c

```diff
@@ -1022,6 +1022,112 @@ err:	API_END_RET_NOTFOUND_MAP(session, ret);
 }
 
 /*
+ * __conn_optrack_dir --
+ *	Set the directory for operation logging
+ */
+static int
+__conn_optrack_dir(WT_SESSION_IMPL *session, const char *cfg[])
+{
+	WT_CONFIG_ITEM cval;
+
+	/* Only use the environment variable if configured. */
+	WT_RET(__wt_config_gets(session, cfg, "use_environment", &cval));
+	if (cval.val != 0 &&
+	    __wt_getenv(session, "WIREDTIGER_OPTRACK",
+			&S2C(session)->optrack) == 0)
+		return (0);
+
+	/*
+	 * If there's no WIREDTIGER_OPTRACK environment variable, use the
+	 * configured database directory.
+	 */
+	S2C(session)->optrack = ".";
+
+	return (0);
+}
+
+/*
+ * A global variable that tells us whether or not we have created a
+ * map file for the current process. Map files, translating binary
+ * numbers into function names are valid per-process, so we have to
+ * set them once per process, not per connection or per session.
+ */
+static bool __wt_optrack_map_setup = 0;
```

Comment:
I'm confused by this. The WT_CONNECTION handle is single-process, so it should be OK to track the field there. I see that it's a recent response to a test failure so I'll dig deeper.

Reply:
@agorrod, this must be an oversight on my part. I probably intended to move this to the connection structure, but forgot. No reason to have it as a static. I will move and retest.